packages/filebrowser/src/listing.ts

```diff
@@ -2106,6 +2165,13 @@ export namespace DirListing {
       } else {
         modified.classList.remove(MODIFIED_COLUMN_HIDDEN);
       }
+
+      if (hiddenColumns?.has?.('file_size')) {
```

Comment:
I don't know why that `?` was added in the previous test. But it should exist if `hiddenColumns` is defined, isn't it?

```suggestion
      if (hiddenColumns?.has('file_size')) {
```

Reply:
Thanks, yeah not sure why it was like that.

I pushed a commit to remove that on the whole file.